<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Application</class>
 <widget class="QMainWindow" name="Application">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>763</width>
    <height>594</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>People</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources/resources.qrc">
    <normaloff>:/icons/main.ico</normaloff>:/icons/main.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QListView" name="container"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>763</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="fileMenu">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="newAction"/>
    <addaction name="openAction"/>
    <addaction name="saveAction"/>
    <addaction name="saveAsAction"/>
    <addaction name="closeAction"/>
   </widget>
   <widget class="QMenu" name="editMenu">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="addAction"/>
    <addaction name="editAction"/>
    <addaction name="deleteAction"/>
    <addaction name="clearAction"/>
   </widget>
   <widget class="QMenu" name="helpMenu">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="aboutAction"/>
    <addaction name="AboutQtAction"/>
   </widget>
   <addaction name="fileMenu"/>
   <addaction name="editMenu"/>
   <addaction name="helpMenu"/>
  </widget>
  <action name="openAction">
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/icons/file/open.ico</normaloff>:/icons/file/open.ico</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="toolTip">
    <string>Open a file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::ApplicationSpecificRole</enum>
   </property>
  </action>
  <action name="saveAction">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/icons/file/save.ico</normaloff>:/icons/file/save.ico</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="toolTip">
    <string>Save the entire table to a file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::ApplicationSpecificRole</enum>
   </property>
  </action>
  <action name="saveAsAction">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/icons/file/save as.ico</normaloff>:/icons/file/save as.ico</iconset>
   </property>
   <property name="text">
    <string>Save As</string>
   </property>
   <property name="toolTip">
    <string>Save the entire table as a new file</string>
   </property>
   <property name="shortcut">
    <string>Alt+S</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::ApplicationSpecificRole</enum>
   </property>
  </action>
  <action name="deleteAction">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/icons/edit/delete.ico</normaloff>:/icons/edit/delete.ico</iconset>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
   <property name="toolTip">
    <string>Deletes the selected item(s) in the table</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::ApplicationSpecificRole</enum>
   </property>
  </action>
  <action name="addAction">
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/icons/edit/add.ico</normaloff>:/icons/edit/add.ico</iconset>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
   <property name="toolTip">
    <string>Creates a new person to the table</string>
   </property>
   <property name="shortcut">
    <string>Alt+A</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::ApplicationSpecificRole</enum>
   </property>
  </action>
  <action name="editAction">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/icons/edit/edit.ico</normaloff>:/icons/edit/edit.ico</iconset>
   </property>
   <property name="text">
    <string>Edit</string>
   </property>
   <property name="toolTip">
    <string>Edits the current selected item in the table</string>
   </property>
   <property name="shortcut">
    <string>Alt+E</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::ApplicationSpecificRole</enum>
   </property>
  </action>
  <action name="clearAction">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/icons/edit/clear.ico</normaloff>:/icons/edit/clear.ico</iconset>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
   <property name="toolTip">
    <string>Deletes all of the items in the table</string>
   </property>
   <property name="shortcut">
    <string>Shift+Del</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::ApplicationSpecificRole</enum>
   </property>
  </action>
  <action name="newAction">
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/icons/file/new.ico</normaloff>:/icons/file/new.ico</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Closes the current list and creates a new one&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::ApplicationSpecificRole</enum>
   </property>
  </action>
  <action name="closeAction">
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/icons/file/close.ico</normaloff>:/icons/file/close.ico</iconset>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
   <property name="toolTip">
    <string>Closes the current file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F4</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::ApplicationSpecificRole</enum>
   </property>
  </action>
  <action name="aboutAction">
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/icons/help/about.ico</normaloff>:/icons/help/about.ico</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::AboutRole</enum>
   </property>
  </action>
  <action name="AboutQtAction">
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/icons/help/qt.ico</normaloff>:/icons/help/qt.ico</iconset>
   </property>
   <property name="text">
    <string>About Qt</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::AboutQtRole</enum>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
